### DISCUSSION TITLE
Pictures in multiple albums are archived if one of the albums is archived

### DISCUSSION THREAD

**Discussion by jaragunde** [ente-io/ente#3047] [Enhancements]
### Description

If a certain picture belongs to albums A and B, where A is archived and B is not, the picture won't appear in the timeline. I think that the expected behavior would be the other way around, the picture would show up in the timeline as long as it belongs to album B and B is not archived.

My use case is: I have an album of an event with many pictures, and I want to select only a few pictures of it for my timeline, and archive the rest; I tried archiving the album and adding the selected pictures to another, unarchived album but that didn't work as expected.

Thanks for taking this in consideration!

### Version

0.9.16

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - Android, Desktop - Linux, Web - Firefox

**Comment by ua741** [comment]
Hey @jaragunde ,

Thanks for the suggestion! I'm converting this GitHub issue into a feature request. 

> Expected behavior would be the other way around, the picture would show up in the timeline as long as it belongs to album B and B is not archived.

We'll consider updating the implementation based on feedback from more users.

**Comment by windmueller** [comment]
I just encountered the same issue. For a birthday party I shared a link so that others could upload their photos. After making a selection of the best ones into a new  album, I wanted to archive the shared album so that it is hidden from my timeline. Unfortunately, all photos are hidden now.

**Comment by huntfx** [comment]
Just gave this a test with uploading 1 image to 2 albums and I agree it's not super intuitive. I don't have as good of a use case but I've noticed different windows/android behaviour.

In Android, hiding one album hides the image from the timeline, but the image is still visible in any other unarchived albums.
In Windows, it does not hide the image from the timeline until both albums are hidden (desired behaviour).

In Android, hiding an album removes it from the list of albums, and adds it to the list of archived albums (desired behaviour).
In Windows, hiding an album marks it as archived but does not remove it from the list of albums, and the archive folder has no album list whatsoever. The workaround suggested in #1571 about archiving shared folders to hide them won't work on Windows.

The browser version and windows seem to match the same behaviour.
